ads/google/a4a/test/test-traffic-experiments.js

```diff
@@ -76,26 +71,26 @@ describe('all-traffic-experiments-tests', () => {
 
     it('handles empty experiments list', () => {
       // Opt out of experiment.
-      sandbox.win.AMP_CONFIG['testExperimentId'] = 0.0;
+      toggleExperiment(sandbox.win, 'testExperimentId', false, true);
```

Comment:
That's exactly the point of this refactoring. We should NOT directly access AMP_CONFIG to read/write experiment status. We should always to it via the methods exposed by `experiments.js`. 

In your unit test here, you don't really care how the experiment is set. You only care if the experiment is on.